data/data/bootstrap/files/usr/local/bin/ironic.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
Is the plan for these to become baremetal platform-specific?

Reply:
Yes, I'll rebase this on https://github.com/openshift-metal3/kni-installer/pull/102